### PR TITLE
Upgrade core version

### PR DESCRIPTION
**Pull request by nlewis84**

### CODE REVIEW

**Comment by redreceipt** [comment]
blocked by https://github.com/ApollosProject/apollos-templates/pull/449

**Comment by nlewis84** [comment]
Okay, @redreceipt, I think this is ready now. Use the heroku CLI to make a local copy of the database. No issues with API or APP that I saw.
Simulator Screen Shot - iPhone 11 - 2021-08-18 at 15 06 43

File: apollos-church-api/package.json [hunk 1]
```diff
@@ -42,9 +42,9 @@
     "randomcolor": "0.5.3"
   },
   "devDependencies": {
-    "@apollosproject/apollo-server-env-mock": "^2.27.2",
-    "@apollosproject/babel-preset-apollos": "^2.27.2",
-    "@apollosproject/eslint-config": "^2.27.2",
+    "@apollosproject/apollo-server-env-mock": "^2.29.3",
```
Comment (redreceipt) [inline apollos-church-api/package.json:45]: why are these on different versions

File: apolloschurchapp/fastlane/Fastfile [hunk 2]
```diff
@@ -45,11 +37,9 @@ platform :ios do
     # get project name
     proj = Dir.glob("../ios/*.xcworkspace")[0][7..-13]
 
-    identifier = product_bundle_id(path: "../ios/" << proj << ".xcodeproj")
-
     app_store_connect_api_key(key_filepath: "ios/apollos.p8", in_house: false)
     setup_ci
-    match(type: 'appstore', app_identifier: [identifier, "#{identifier}.OneSignalNotificationServiceExtension"], readonly: true)
```
Comment (redreceipt) [inline apolloschurchapp/fastlane/Fastfile:52]: you'll have to add the one signal extension bundle ID to the match app identifier env variable now. it's comma separated, no space

File: apollos-church-api/package.json (hunk 1 above)
Comment (nlewis84) [reply apollos-church-api/package.json:45]: Release 2.29.3 changed them

File: apolloschurchapp/fastlane/Fastfile (hunk 2 above)
Comment (nlewis84) [reply apolloschurchapp/fastlane/Fastfile:52]: So this shouldn't have been merged? Does this change look right?

Comment (redreceipt) [reply apolloschurchapp/fastlane/Fastfile:52]: That change is correct, we just specify that variable as an env now

File: apollos-church-api/package.json (hunk 1 above)
Comment (redreceipt) [reply apollos-church-api/package.json:45]: No i mean why didn't they get bumped to 2.30

Comment (nlewis84) [reply apollos-church-api/package.json:45]: Latest version of the template also has them at 2.29.3. Not sure why.

File: apolloschurchapp/fastlane/Fastfile (hunk 2 above)
Comment (nlewis84) [reply apolloschurchapp/fastlane/Fastfile:52]: Oh gotcha

File: apollos-church-api/package.json (hunk 1 above)
Comment (redreceipt) [reply apollos-church-api/package.json:45]: Oh weird

**Review by redreceipt — APPROVED** [review]

**Pull request merged**